tools/create_certs/rootCA.srl

```diff
@@ -0,0 +1 @@
+87F5115738EE8D42
```

Comment:
How did this serial number get chosen?  Do we want all certs to have it?

Reply:
Good point -- I think it is unnecessary for the demo. Thanks!